packages/ui-components/src/components/assistant-ui/markdown-text/markdown-text.tsx

```diff
@@ -1,5 +1,3 @@
-import '@assistant-ui/react-markdown/styles/dot.css';
-
```

Comment:
the rendered their default spinner. This is problematic as it does not render after tool calls, even though conversation is not finished